### PR TITLE
[Proposition] Allow user to reset setting to default values

### PR DESCRIPTION
**Pull request by MangelMaxime**
When reporting issues I always have a hard top finding the correct configuration because I never know which settings changed or not.

Adding this button allows the user to restore the default settings of Fantomas to start from a blank page

### CODE REVIEW

**Comment by dawedawe** [comment]
Love it. Thanks a lot!

**Comment by MangelMaxime** [comment]
I agree that having it at the top of the setting would be better.

But I am struggling with finding a place that feels nice for it.

Right now the one that feels the least out of place is this one:

CleanShot 2023-12-18 at 09 47 32@2x

**Comment by dawedawe** [comment]
That would be fine for me :)

**Review by dawedawe — APPROVED** [review]

File: src/client/src/styles/style.css [hunk 1]
```diff
@@ -866,3 +866,11 @@ nav > div #docs-btn .long-text {
         color: var(--light);
     }
 }
+
+.reset-settings {
```
Comment (nojaf) [inline src/client/src/styles/style.css:870]: Please move this closer to the `/* Fantomas tab */`

File: src/client/fsharp/FantomasOnline/View.fs [hunk 2]
```diff
@@ -341,13 +341,26 @@ let settings isFsi model dispatch =
...
+            ]
+
         fragment [] [
             VersionBar.versionBar (sprintf "Version: %s" model.Version)
             fantomasMode
             fileExtension
             hr []
             searchBox
             options
+            if userChangedSettings model then
```
Comment (nojaf) [inline src/client/fsharp/FantomasOnline/View.fs:358]: Yes, please move it up indeed.

**Review by nojaf — CHANGES_REQUESTED** [review]
I'm ok with having this.
Please move it higher though.

**Review by nojaf — APPROVED** [review]
Great stuff, thanks @MangelMaxime!

**Pull request merged**